config/git.php

```diff
@@ -17,6 +17,20 @@
 
     'enabled' => env('STATAMIC_GIT_ENABLED', false),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Queue Connection
+    |--------------------------------------------------------------------------
+    |
+    | You may choose which queue connection should be used when dispatching
+    | commit jobs. Unless specified, the default connection will be used.
+    |
+    | https://statamic.dev/git-integration#queueing-commits
+    |
+    */
+
+    'queue' => env('STATAMIC_GIT_QUEUE_CONNECTION'),
```

Comment:
This is a total nitpick, but move this one block down between 'Automatically Run' and 'Dispatch Delay', so that all the queue config is next to each other 👍